README.md

```diff
@@ -1 +1,72 @@
-# sc-templates
+# Dragonchain Smart Contract Templates
+
+* **Overview**
+  - Design goals
+  - Smart contract (SC)
+  - Getting started
+
+* **Design goals**
+  - Scalability and Flexibility - Developers can use any programming languages they would like as long as they can configure a dockerfile correctly. Developers can develop up to 20 smart contract but can request for custom increase if needed.
```

Comment:
Developers can create up to 20 smart contract to start, and can request for custom increase if needed.